DQM/Phase2OuterTracker/interface/OuterTrackerMonitorTTCluster.h

```diff
@@ -0,0 +1,61 @@
+#ifndef Phase2OuterTracker_OuterTrackerMonitorTTCluster_h
+#define Phase2OuterTracker_OuterTrackerMonitorTTCluster_h
+
+#include <vector>
+#include <memory>
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "DQM/SiStripCommon/interface/TkHistoMap.h"
```

Comment:
hi @lmoreels - while waiting for @rekovic to review - I had some comments

- here i guess <vector>, <memory> and TkHistoMap.h includes are not needed. Please remove them (maybe they are needed in the cc file, I didn't check). On the other hand, include <string> is needed